### PR TITLE
Archives, uc center plugin, and single posts

### PR DESCRIPTION
**Pull request by ghost**
#reviewmerge @jeremyfelt @philcable Create templates and styles for archives, uc center people and organizations, and single posts. 

Would one of you two help me get the category title to show up above the uc center people and organization archive pages?

### CODE REVIEW

File: articles/post-wsuwp_uc_entity.php [hunk 1]
```diff
@@ -0,0 +1,29 @@
+<article id="post-<?php the_ID(); ?>" <?php post_class(); ?>>
+
+	<header class="article-header">
+		<hgroup>
+			<?php if ( is_single() ) : ?>
+				<h1 class="article-title"><?php the_title(); ?></h1>			<?php else : ?>
```
Comment (philcable) [inline articles/post-wsuwp_uc_entity.php:6]: This `<?php else: ?>` isn't necessary if you don't need to output anything for archive views. And if that is indeed the case, you might consider moving the `is_single()` check outside of the `<header>` element (example below) so we're not outputting empty markup.

```
<?php if ( is_single() ) : ?>
<header class="article-header">
...
</header>
<?php endif; ?>
```

File: articles/post-wsuwp_uc_entity.php [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+			the_excerpt();
+			?>
+		</div><!-- .article-summary -->
+	<?php else : ?>
+		<div class="article-body">
+			<?php the_content(); ?>
+			<?php wp_link_pages( array( 'before' => '<div class="page-links">' . __( 'Pages:', 'spine' ), 'after' => '</div>' ) ); ?>
+		</div>
+	<?php endif; ?>
+
```
Comment (philcable) [inline articles/post-wsuwp_uc_entity.php:28]: I think we're missing the closing tag for the `.flex-body` div here.

File: articles/post-wsuwp_uc_person.php [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+
+					if ( ! empty( $display_data['title'] ) ) : ?><div class="person-title"><?php echo esc_html( $display_data['title'] ); ?></div><?php endif;
+					if ( ! empty( $display_data['title_secondary'] ) ) : ?><div class="person-title-secondary"><?php echo esc_html( $display_data['title_secondary'] ); ?></div><?php endif;
+					if ( ! empty( $display_data['email'] ) ) : ?><div class="person-email"><a href="mailto:<?php echo esc_html( $display_data['email'] ); ?>"><?php echo esc_html( $display_data['email'] ); ?></a></div><?php endif;
+
+
+				} ?>
+			</hgroup>
+		</header>
+</div>
```
Comment (philcable) [inline articles/post-wsuwp_uc_person.php:43]: Not a big deal, but adjusting the indents here would make it easier to tell if all the HTML tags are properly nested and closed.

File: parts/single-layout-wsuwp_uc_person.php [hunk 4]
```diff
@@ -0,0 +1,48 @@
+<?php while ( have_posts() ) : the_post(); ?>
+<section class="row single gutter pad-top pad-bottom">
+
+	<div class="column one">
+			<h1 class="article-title"><?php the_title(); ?></h1>
```
Comment (philcable) [inline parts/single-layout-wsuwp_uc_person.php:5]: Consider adjusting the indents for readability in this file, too.

**Review by philcable — COMMENTED** [review]
Looking good! I left a couple comments about some very minor things. I'll let @jeremyfelt take a quick look through it too before merging just in case I missed anything.

**Review by jeremyfelt — APPROVED** [review]

**Pull request merged**